zzk/service/hoststate_test.go

```diff
@@ -33,27 +33,23 @@ var (
 	ErrTestNoAttach = errors.New("could not attach to container")
 )
 
+const (
+	servicePath string = "/services/serviceid"
+	hostPath    string = "/hosts/hostid"
+	hostId      string = "hostid"
+	serviceId   string = "serviceid"
+	serviceName string = "serviceA"
+	imageId     string = "imageid"
+	containerId string = "containerid"
+)
+
 // Test Case: Bad state id
 func (t *ZZKTest) TestHostStateListener_Spawn_BadStateID(c *C) {
-	// Pre-requisites
-	conn, err := zzk.GetLocalConnection("/")
-	c.Assert(err, IsNil)
-	handler := &mocks.HostStateHandler{}
 
-	// Basic set up
-	svc := &service.Service{
-		ID:     "serviceid",
-		Name:   "serviceA",
-		PoolID: "poolid",
-	}
-	spth := "/services/serviceid"
-	sdat := &ServiceNode{Service: svc}
-	err = conn.Create(spth, sdat)
-	c.Assert(err, IsNil)
-	err = conn.CreateDir("/hosts/hostid")
-	c.Assert(err, IsNil)
+	conn := setUpServiceAndHostPaths(c)
```

Comment:
I made this change not as a convenience, but to improve readability. This change reduces the amount of code that the user has to skip past to get to the interesting part of the test. It also reduces the number of places we'll need to change if we need to modify the test setup. Additionally, this change reduced the size of each test function by several lines (as mentioned above, that improves readability), and reduced the overall file size by ~120 lines.